apps/docs/src/app/core/component-docs/card/examples/card-example.component.html

```diff
@@ -48,7 +48,7 @@ <h2 fd-card-title>Card Title</h2>
 <h3>Card header with avatar, subtitle, counter and badge</h3>
 <fd-card badge="New" cardType="list">
     <fd-card-header>
-        <fd-avatar image="http://lorempixel.com/400/400/nature/4/"></fd-avatar>
+        <fd-avatar image="http://picsum.photos/id/1018/400/4/"></fd-avatar>
```

Comment:
`http://picsum.photos/id/1018/400/4/` I think you should remove `/4/` part. Its setting the image height to 4px 😋